decidim-dev/lib/decidim/dev/test/rspec_support/helpers.rb

```diff
@@ -39,6 +39,18 @@ def expect_user_logged
   def have_admin_callout(text)
     have_selector(".callout--full", text:)
   end
+
+  def do_stub_rq_with_rs_format(rq_url, rs_format)
```

Comment:
The name of this method is still bothering me a bit since it has a bit too many abbreviations to my personal liking.

Could we do something like `stub_get_request_with_format(url, format)`, it would be a bit more readable?